core/identity-hub-did/src/main/java/org/eclipse/edc/identityhub/did/DidServicesExtension.java

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.identityhub.did;
+
+public class DidServicesExtension {
```

Comment:
`implements ServiceExtension` ?

Reply:
yeah, it's a stub at the moment. implementation will follow later